WakeLockSentinel/WakeLockSentinel_onrelease_EventHandler.html

```diff
@@ -0,0 +1,49 @@
+<div>{{securecontext_header}}{{APIRef("Screen Wake Lock API")}}</div>
+
+<p class="summary">The <strong><code>onrelease</code></strong> property of the {{domxref("WakeLockSentinel")}} is fired when the sentinel object's handle has been released.</p>
+
+<p>A {{domxref("WakeLockSentinel")}} can be released manually via the {{domxref('WakeLockSentinal.release','release()')}} method, or automatically via the platform wake lock. This can happen if the document becomes inactive or looses visibility, if the device is low on power or the user turns on a power save mode.</p>
+
+<p>This interface inherits from the {{domxref("Event")}} interface.</p>
+
+<p>{{InheritanceDiagram(700, 60, 20)}}</p>
```

Comment:
Here's another place where a JSON file has to be updated. Can you please submit a PR to add `Release` to this file: https://github.com/mdn/kumascript/blob/master/macros/InterfaceData.json